src/collection/dimensions.js

```diff
@@ -925,6 +951,21 @@ defineDimFns( {
   name: 'height'
 } );
 
+elesfn.padding = function(){
+  if(this.length > 0) {
+    var ele = this[0];
+    var _p = ele._private;
+    if( _p.autoPadding !== undefined ) {
```

Comment:
This check isn't safe.  A node can change from parent to non-parent and the value would still be defined